### DISCUSSION TITLE
Manage HomeSpan in HomeKit and Home Assistant

### DISCUSSION THREAD

**Discussion by G-D-D-971** [HomeSpan/HomeSpan#847] [Q&A]
Hello, at home, I have two complementary home automation systems, HomeKit, which works perfectly well and has functions basic enough to allow the whole family to use it, and the other Home Assistant that allows me to put control screens for my home accessories with old tablets (it avoids looking for your phone to turn off the lights...). The fact is that for the moment, all the accessories I have go through Home Assistant to be added to HomeKit where the others have a Home Assistant integration with which to dialogue, but for HomeSpan, I don't see anything... Does anyone have an idea?@HomeSpan ?

**Comment by HomeSpan** [comment]
Sorry, HomeSpan does not include anything related to Home Assistant.  However, HomeSpan operates exactly like any HomeKit Accessory, so if you can add a generic HomeKit device to Home Assistant, you should be able to the same with a HomeSpan device.   I have not used Home Assistant so can't really weigh-in on the best method to integrate.
